.github/CODEOWNERS

```
@@ -2,3 +2,5 @@
 # Each line is a file pattern followed by one or more owners.
 
 *                                   @MetaMask/wallet-framework-engineers
+!docs/**
+!examples/**
```

The natural minimum protection measure would be to add @MetaMask/engineering as a codeowner so that only internal engineers can approve a PR, but I understand that this is not desirable due to the excessive notifications it would send out.

```suggestion
!docs/**  @MetaMask/engineering 
!examples/**  @MetaMask/engineering 
```